src/services/AdvancedTradingView/store/advancedTradingView.reducer.ts

```diff
@@ -58,6 +65,71 @@ const advancedTradingViewSlice = createSlice({
         },
       }
     },
+
+    setSwapsBurnsAndMintsDataForAllPairs: (
+      state: InitialState,
+      action: PayloadAction<Array<SetSwapsBurnsAndMintsActionPayload>>
+    ) => {
+      const updatedAdapters: AdapterType = {
+        ...state.adapters,
+      }
+
+      action.payload.forEach(adapter => {
+        const { key, pairId, data, hasMore } = adapter
+
+        const nextPairSwapData = state.adapters[key][pairId]?.[AdapterPayloadType.SWAPS]?.data ?? []
+        const nextPairBurnsAndMintsData = state.adapters[key][pairId]?.[AdapterPayloadType.BURNS_AND_MINTS]?.data ?? []
+
+        data.swaps.forEach(
+          element => !nextPairSwapData.some(el => el.id === element.id) && nextPairSwapData.push(element)
+        )
+        data.burnsAndMints.forEach(
+          element =>
+            !nextPairBurnsAndMintsData.some(el => el.id === element.id) && nextPairBurnsAndMintsData.push(element)
+        )
```

Comment:
Can we have a different name for `element` and `el`? Just to have a better reading in this code

Reply:
Agree, this is updated. 👍🏼